jujugui/templates/index.html.mako

```diff
@@ -465,6 +465,6 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
     }
     </script>
     <!-- End Google Tag Manager -->
-
+    <script type="text/javascript" src="https://js.stripe.com/v2/"></script>
```

Comment:
The instantiation of some of the code included above here requires this aiui so putting it down here means that it's possible that it won't be available when needed.